lib/WebpackConfig.js

```diff
@@ -222,6 +237,18 @@ class WebpackConfig {
         this.useReact = true;
     }
 
+    enableTypeScriptLoader(options = {}) {
+        this.useTypeScriptLoader = true;
+
+        for (const optionKey of Object.keys(options)) {
+            if (!(optionKey in this.typeScriptOptions)) {
+                throw new Error(`Invalid option "${optionKey}" passed to enableTypeScriptLoader(). Valid keys are ${Object.keys(this.typeScriptOptions).join(', ')}`);
+            }
```

Comment:
I usually like adding nice validation like this, but I'm afraid maintaining this will be a pain (we need to add a new option whenever they add a new option). The typescript-loader should ideally validate these keys themselves.